templates/ra_guac_autoscale_public_alb.template.cfn.yaml

```diff
@@ -0,0 +1,623 @@
+  Version: 0.3.1
+  cfn-lint:
+    config:
+      ignore_checks:
+        - E7001
+        - W1011
+Outputs:
+  AlbSecurityGroupId:
+    Description: Security Group ID for the Guacamole Application Load Balancer
+    Value: !Ref AlbSecurityGroup
+  Ec2SecurityGroupId:
+    Description: Security Group ID for Guacamole instances
+    Value: !Ref Ec2SecurityGroup
+  LoadBalancerDns:
+    Description: DNS name for the ALB
+    Value: !Sub ${ALB.DNSName}
+  LoadBalancerName:
+    Description: Name of the ALB
+    Value: !Ref ALB
+  TargetGroupName:
+    Description: Name of the ALB Target Group
+    Value: !Ref ALBTargetGroup
+Parameters:
+  AmiId:
+    Default: ''
+    Description: (Optional) AMI ID -- will supersede Lambda-based AMI lookup using AmiNameSearchString
+    Type: String
+  AmiNameSearchString:
+    Default: amzn-ami-hvm-2017.09*.2017*-x86_64-gp2
+    Description: Search pattern to match against an AMI Name
+    Type: String
+  BrandText:
+    Default: ''
+    Description: Text/Label to display branding for the Guac Login page
+    Type: String
+  DesiredCapacity:
+    Default: '1'
+    Description: The number of instances the autoscale group will spin up initially
+    MinLength: '1'
+    Type: String
+  ForceUpdateToggle:
+    AllowedValues:
+      - A
+      - B
+    Default: A
+    Description: A/B toggle that forces a change to a LaunchConfig property, triggering the AutoScale Update Policy
+    Type: String
+  GuacBaseDN:
+    Default: CN=GuacConfigGroups
+    Description: The base of the DN for all Guacamole configurations.
+    Type: String
+  GuacamoleVersion:
+    Default: 1.0.0
+    Description: Guacamole version tag. Defaults to 1.0.0
+    Type: String
+  GuacdVersion:
+    Default: 1.0.0
+    Description: Guacd version tag. Defaults to 1.0.0
+    Type: String
+  InstanceType:
+    AllowedValues:
+      - t2.micro
+      - t2.small
+      - t2.medium
+      - t2.large
+      - t2.xlarge
+      - t2.2xlarge
+      - t3.micro
+      - t3.small
+      - t3.medium
+      - t3.large
+      - t3.xlarge
+      - t3.2xlarge
+      - c5.large
+      - c5.xlarge
+      - c5.2xlarge
+      - c5.4xlarge
+      - m5.large
+      - m5.xlarge
+      - m5.2xlarge
+      - m5.4xlarge
+      - r4.large
+      - r4.xlarge
+    Default: t2.micro
+    Description: Amazon EC2 instance type for the Guacamole Instances
+    Type: String
+  KeyPairName:
+    Description: Public/private key pairs allow you to securely connect to your instance after it launches
+    Type: 'AWS::EC2::KeyPair::KeyName'
+  LdapDN:
+    Default: 'DC=ad,DC=example,DC=com'
+    Description: Distinguished Name (DN) of the LDAP directory
+    MinLength: '1'
+    Type: String
+  LdapServer:
+    Default: ad.example.com
+    Description: Name of LDAP server Guacamole will authenticate against
+    MinLength: '1'
+    Type: String
+  MaxCapacity:
+    Default: '2'
+    Description: The maximum number of instances for the autoscale group
+    MinLength: '1'
+    Type: String
```

Comment:
more of a nit but this should probably be the `Number` type

Reply:
changed to `Number` type